spdxexp/license.go

```diff
@@ -958,13 +1172,18 @@ func licenseRanges() [][][]string {
 		},
 		{
 			{
-				"MPL-1.0",
```

Comment:
Is there a reason to delete these MPL entries?

Reply:
Hi,
The MPL range entries are not deleted, they are still there.

[https://github.com/github/go-spdx/blob/9af17522cc6e02dabedb25f9d28b3ca69677f9cc/spdxexp/license.go#L1167](https://github.com/github/go-spdx/blob/9af17522cc6e02dabedb25f9d28b3ca69677f9cc/spdxexp/license.go#L1167)